tests/components/blebox/test_binary_sensor.py

```diff
@@ -0,0 +1,45 @@
+"""Blebox binary_sensor entities test."""
+from unittest.mock import PropertyMock
+
+import blebox_uniapi
+import pytest
+
+from homeassistant.components.binary_sensor import BinarySensorDeviceClass
+from homeassistant.const import ATTR_DEVICE_CLASS, STATE_ON
+from homeassistant.helpers import device_registry as dr
+
+from .conftest import async_setup_entity, mock_feature
+
+
+@pytest.fixture(name="rainsensor")
+def airsensor_fixture():
```

Comment:
As you know, I am a big fan of type hints, even in tests.
It was not compulsory for this PR, but it would be great to add here

Reply:
Type hints should be in same PR with sensor description refactor?

Reply:
Smaller PRs are easier to review. I suggest to split from sensor description refactor.